cime_config/acme/machines/config_compilers.xml

```diff
@@ -1120,14 +1116,4 @@ for mct, etc.
 	<ALBANY_PATH>$(ALBANY_PATH)</ALBANY_PATH>
 </compiler>
 
-<compiler MACH="caldera">
```

Comment:
Ah, sorry. I'll put it back. It was the only block that defined MACH and not COMPILER and I didn't want to support that, but I already had to make change to support it and just forgot to restore this.